__tests__/app.test.js

```diff
@@ -67,7 +67,37 @@ describe("app!!", ()=>{
           expect(sortedArticlesKeys).toEqual(expect.arrayContaining(['article_id', 'comment_count']))
         })
       })
+      //            TASK 6
+        test('should return comments if there is any',()=>{
```

Comment:
This isn't a huge deal but ideally each different end point would be in it's own describe block to separate them out 🎨